lms/static/scripts/frontend_apps/components/InstructorToolbar.tsx

```diff
@@ -70,7 +71,11 @@ export default function InstructorToolbar() {
         </h2>
       </div>
 
-      {withGradingControls ? <GradingControls students={students} /> : <div />}
+      {withGradingControls ? (
+        <GradingControls students={students} scoreMaximum={scoreMaximum} />
+      ) : (
+        <div />
```

Comment:
What does the empty `<div>` do here? Should this be `{withGradingControls && ...}` instead?